blockchain/contracts/ProofOfPhysicalAddress.sol

```diff
@@ -22,10 +26,11 @@ contract ProofOfPhysicalAddress {
         uint256 confirmationBlock;
     }
 
-    function ProofOfPhysicalAddress() public
+    function ProofOfPhysicalAddress(address _registry) public
     {
         owner = msg.sender;
         signer = owner;
+        registry = EthereumClaimsRegistryInterface(_registry);
```

Comment:
Maybe also add a method to change `registry`?

Reply:
will do!

Reply:
done